### PR TITLE
EPMRPP-79221 || Default Defect Type color on group select fix, EPMRPP-79165 || Color picker background color fix

### PR DESCRIPTION
**Pull request by tr1ble**

### CODE REVIEW

**Comment by Vadim73i** [comment]
Are you sure you fixed "Selector in color picker is bigger than on Mock-ups"?

**Comment by tr1ble** [comment]
> Are you sure you fixed "Selector in color picker is bigger than on Mock-ups"?

I set size of it accroding to Mock-up long time ago

**Review by Vadim73i — APPROVED** [review]

File: app/src/components/main/hexColorPicker/hexColorPicker.scss [hunk 1]
```diff
@@ -57,6 +58,10 @@
   color: $COLOR--almost-black;
 }
 
+section {
+  background-color: $COLOR--white-two;
+}
```
Comment (chivekrodis) [inline app/src/components/main/hexColorPicker/hexColorPicker.scss:63]: Please make sure that solution doesn't affected each section in the app

**Review by AmsterGet — CHANGES_REQUESTED** [review]

File: app/src/pages/inside/projectSettingsPageContainer/content/defectTypes/modals/addEditDefectTypeModal/addEditDefectTypeModal.jsx [hunk 2]
```diff
@@ -220,6 +223,11 @@ const AddEditDefectTypeModal = ({
           name={COLOR_FIELD_KEY}
           className={cx('color-picker')}
           dataAutomationId={COLOR_FIELD_KEY + FIELD}
+          onChange={(value) => {
+            change(COLOR_FIELD_KEY, value);
```
[the remark this answers is not in the document]
Comment (tr1ble) [reply app/src/pages/inside/projectSettingsPageContainer/content/defectTypes/modals/addEditDefectTypeModal/addEditDefectTypeModal.jsx:227]: I'be tried without it, but it didn't work for some reason. I'ok try again

**Review by AmsterGet — APPROVED** [review]

**Pull request merged**